specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.5-preview.1/backuprestore.json

```diff
@@ -352,11 +352,15 @@
         "token": {
           "type": "string",
           "description": "The SAS token pointing to an Azure Blob storage container"
+        },
+        "useManagedIdentity": {
+          "type": "boolean",
+          "default": false,
+          "description": "Indicates which authentication method should be used. If set to true, Managed HSM will use the configured user-assigned managed identity to authenticate with Azure Storage. Otherwise, a SAS token has to be specified."
         }
       },
       "required": [
-        "storageResourceUri",
-        "token"
```

Comment:
This is a breaking change even when changing the api-version. See https://aka.ms/AzBreakingChangesPolicy (internal only). What happens when a client sends `null` or an empty string? That's likely what is going to happen. Can you handle that on the backend to treat it the same as absent?

@mikekistler @JeffreyRichter @johanste does this need to go through the breaking change approval process, or will that depend on their answer? We can, if needed, patch this in code generated clients, but most languages are hand-coding re-export of models so even that may not be necessary.

Reply:
From the service team,

> When the useManagedIdentity is set, the token is optional and will be ignored at the service side.